lms/envs/common.py

```diff
@@ -3664,3 +3664,6 @@ def _make_locale_paths(settings):
 ASSETS_ACCESS_BY_TOKEN = True
 ASSETS_TOKEN_ENCRYPTION_KEY = 'secret'
 ASSETS_TOKEN_TTL = 600
+
+USE_X_FORWARDED_PORT = True
+RESPECT_X_FORWARDED_HEADERS = True
```

Comment:
I think we should only set this `RESPECT_X_FORWARDED_HEADERS`, because it's impact is known and limited. (https://github.com/edx-solutions/edx-platform/blob/942cbb2493a5023a19ed833f92cdaba45a47d8b6/common/djangoapps/third_party_auth/strategy.py#L65-L85) 

Setting `USE_X_FORWARDED_PORT` true might have some unknown implications.